app/model/commands/PublishAtomCommand.scala

```diff
@@ -141,11 +147,41 @@ case class PublishAtomCommand(id: String, override val stores: DataStores, youTu
     }
   }
 
+  private def parseDescriptionForYoutube(description: String): String = {
+      val html = Jsoup.parse(description)
+      html.select("a").remove()
+      html.text()
+  }
+
+  private def getComposerLinkText(atomId: String): String = {
+
+    try {
+      val usages: JsValue = (capi.capiQuery("/atom/media/" + atomId + "/usage", true) \ "response" \ "results").get
+      val usagesList = usages.as[List[String]]
+
+      val composerPage = usagesList.find(usage => {
+        val query = capi.capiQuery(usage, true)
+        val contentType = (capi.capiQuery(usage, true) \ "response" \ "content" \ "type").get.as[String]
+        contentType == "video"
+      })
+
+      composerPage match {
+        case Some(page) => "\n View the video at https://www.theguardian.com/" + page
+        case None => ""
+      }
+    }
+  }
+
   private def updateYoutubeMetadata(previewAtom: MediaAtom, asset: Asset) = {
+
+    val description = previewAtom.description.map(description => {
```

Comment:
As we discussed, this becomes difficult because getting the youtube description requires querying capi so I'm just going to leave this here.